### ISSUE TITLE
Abandoned connection from default session

### ISSUE THREAD

**Issue by FrantaM** [pponec/ujorm#1]
`OrmHandler` uses default session to create database yet this session is never closed and the connection is abandoned.

https://github.com/pponec/ujorm/blob/1046810017417dff0960f02bf472f282873adfcb/project-m2/ujo-orm/src/main/java/org/ujorm/orm/OrmHandler.java#L263-L265

```
org.apache.tomcat.dbcp.pool2.impl.DefaultPooledObject$AbandonedObjectCreatedException: Pooled object created 2019-03-27 09:30:56 +0100 by the following code has not been returned to the pool:
	at org.apache.tomcat.dbcp.pool2.impl.DefaultPooledObject.allocate(DefaultPooledObject.java:194)
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:453)
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:359)
	at org.apache.tomcat.dbcp.dbcp2.PoolingDataSource.getConnection(PoolingDataSource.java:134)
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.getConnection(BasicDataSource.java:1543)
	at cz.sa.ares.database.ujo.DataSourceDialectDelegate.createConnection(DataSourceDialectDelegate.java:93)
	at org.ujorm.orm.metaModel.MetaDatabase.createConnection(MetaDatabase.java:470)
	at org.ujorm.orm.Session.getConnection_(Session.java:974)
	at org.ujorm.orm.Session.getConnection(Session.java:1016)
	at org.ujorm.orm.metaModel.MetaDbService.create(MetaDbService.java:70)
	at org.ujorm.orm.metaModel.MetaDatabase.create(MetaDatabase.java:385)
	at org.ujorm.orm.OrmHandler.loadDatabase(OrmHandler.java:264)
	at org.ujorm.orm.OrmHandler.loadDatabase(OrmHandler.java:211)
        ...

```

**Comment by FrantaM** [comment]
Fixed in c0b976ff22023320793f21305b7fe59562d98500.

**Comment by pponec** [comment]
Starting with Ujorm 1.91, you will need to modify a database connection configuration for H2 database in-memory. The reasons are:

> By default, closing the last connection to a database closes the database. For an in-memory database, this means the content is lost. To keep the database open, add ;DB_CLOSE_DELAY=-1 to the database URL. To keep the content of an in-memory database as long as the virtual machine is alive, use jdbc:h2:mem:test;DB_CLOSE_DELAY=-1. 

See the original [documentation](http://www.h2database.com/html/features.html)

An alternative solution is to modify the Ujorm configuration as shown:

```java
        boolean yesIWantToChangeDefaultParameters = true;
        if (yesIWantToChangeDefaultParameters) {
            MetaParams params = new MetaParams();
            params.set(MetaParams.AUTO_CLOSING_DEFAULT_SESSION, false);
            result.config(params);
        }

```
